cmake/Common.cmake

```
@@ -28,6 +28,13 @@ foreach(p LIB BIN INCLUDE DATA)
   endif()
 endforeach()
 
+
+
+# -----------------------------------------------------------------------------
+# CPP11 
+# -----------------------------------------------------------------------------
+set (CMAKE_CXX_STANDARD 11)
+
```

Also add
```suggestion
set (CMAKE_CXX_STANDARD_REQUIRED TRUE)

```
to avoid fallback to previous standard version if C++11 is not available (see https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD_REQUIRED.html#prop_tgt:CXX_STANDARD_REQUIRED)

---

thanks!